src/summary/documentation/dimensions-card.component.tsx

```diff
@@ -30,7 +30,7 @@ export default function DimensionsCard(props: DimensionsCardProps) {
     <SummaryCard
       name="Height & Weight"
       match={props.match}
-      styles={{ flex: 1, margin: ".5rem", maxWidth: "46rem" }}
+      styles={{ margin: ".5rem", width: "45.5rem", maxWidth: "46rem" }}
```

Comment:
@jonathandick will this be responsive across different screen sizes? Setting the width explicitly might cause unwanted behavior on smaller screens.

Reply:
In my tests, when converting to Iphone, or Ipad in chrome, it seems to be responsive despite the fixed width property. Do you have a suggested alternative approach to this? I could use the css guidance on responsiveness. 

![image](https://user-images.githubusercontent.com/139934/69880988-618d8580-1280-11ea-975b-6cb72c7625c2.png)